packages/squiggle-lang/src/rescript/utility/E.res

```diff
@@ -324,7 +325,7 @@ module A = {
         }
     )
   let filter = (o, e) => Js.Array.filter(o, e)
```

Comment:
while we're at it maybe we'll change line 326 to `let filter = Js.Array.filter`?